src/conn/conn_open.c

```diff
@@ -118,7 +118,8 @@ __wti_connection_close(WT_CONNECTION_IMPL *conn)
     __wt_blkcache_destroy(session);
 
     /* Shut down layered table manager - this should be done after closing out data handles. */
-    WT_TRET(__wt_layered_table_manager_destroy(session, true));
+    if (__wt_conn_is_disagg(session))
```

Comment:
Nitpick, it might make more sense to move this check into `__wt_layered_table_manager_destroy` to keep this function a "straight line" and to prevent other potential callers from worrying about when they can do this.

Reply:
done